nemo/lightning/pytorch/callbacks/memory_profiler.py

```diff
@@ -0,0 +1,78 @@
+import os
+
+import torch
+from pytorch_lightning.callbacks.callback import Callback
+from torch.utils.viz._cycles import warn_tensor_cycles
```

Comment:
Can we move this import inside the callback to be safe?

Reply:
Isn't it better to fail early if there is an issue with this import than fail inside the callback?